src/main/java/com/google/sps/app/servlets/NewEventServlet.java

```diff
@@ -0,0 +1,111 @@
+
+package com.google.sps.app.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.KeyFactory;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.gson.Gson;
+import com.google.sps.model.Category;
+import com.google.sps.model.Event;
+import com.google.sps.util.UUIDs;
+import java.io.IOException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Date;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.commons.text.StringEscapeUtils;
+
+import io.opencensus.common.ServerStatsFieldEnums.Id;
+
+@WebServlet("/NewEvent")
+public class NewEventServlet extends HttpServlet {
+
+    private static String TITLE_PARAM = "text-input-title";
+    private static String ESTIMATED_COST_PARAM = "text-input-estimatedCost";
+    private static String DATE_COST_PARAM = "text-input-date";
```

Comment:
Should this be just DATE_PARAM? Same with LOCATION_PARAM

Reply:
oh my bad